configure.ac

```diff
@@ -194,4 +207,4 @@ AC_MSG_NOTICE([Summary of build options:
   Library types:      Shared=${enable_shared}, Static=${enable_static}
   LDAP:               ${with_ldap}
   Challenge-Response: ${with_cr}
-])
+])
```

Comment:
There an unnecessary (and wrong) change here removing the linefeed at the end of input, I don't want that in.